ci-operator/config/tnozicka/openshift-acme/master.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
@stevekuznetsov just as a place for feedback - YAML would be much nicer for user experience (look, trailing commas, ...) and you could write comments there

Reply:
Also good job with the declarative pipeline flow, I like it! Took me a while to figure how to do builds close to how users would (with the multi-stage injection build trick), but that's mostly docs issue I guess. Would be awesome to be able to inject it into s2i directly but I can see there are more important things to tackle ;)

Reply:
We are about a couple days from having YAML :) @petr-muller will deliver that for you

Reply:
We have a JIRA ticket for s2i support: https://jira.coreos.com/browse/DPTP-108

Reply:
Can you open an issue on our JIRA to add documents to the ci-operator repo on multi-stage support, please?